tests/Functional/ContentEventRenderTest.php

```diff
@@ -183,42 +167,510 @@ public function testEventRegistrationDateDescription(): void {
+
+    $practical_list_content = $this->assertSession()->elementExists('css', 'dl.ecl-description-list', $practical_content);
+    $field_list_assert = new FieldListAssert();
+    $field_list_expected_values = [
+      'items' => [
+        [
+          'label' => 'When',
+          'body' => 'Wednesday 19 February 2020, 01:00',
+        ], [
+          'label' => 'Languages',
+          'body' => 'Estonian, French',
+        ],
+      ],
+    ];
+    $field_list_html = $practical_list_content->getOuterHtml();
+    $field_list_assert->assertPattern($field_list_expected_values, $field_list_html);
+    $field_list_assert->assertVariant('horizontal', $field_list_html);
+
+    // Check case when start and end dates are different.
+    $start_date = (clone $static_time)->modify('+ 1 days');
+    $end_date = (clone $static_time)->modify('+ 10 days');
+    $node->set('oe_event_dates', [
+      'value' => $start_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+      'end_value' => $end_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+    ])->save();
+    $this->drupalGet($node->toUrl());
+
+    $icons_text_expected_values = [
+      'items' => [
+        [
+          'icon' => 'file',
+          'text' => 'Financing',
+        ], [
+          'icon' => 'calendar',
+          'text' => "19 February 2020, 01:00\n to 28 February 2020, 01:00",
+        ],
+      ],
+    ];
+    $icons_text_assert->assertPattern($icons_text_expected_values, $details_list_content->getOuterHtml());
+
+    $field_list_expected_values = [
+      'items' => [
+        [
+          'label' => 'When',
+          'body' => "Wednesday 19 February 2020, 01:00\n to Friday 28 February 2020, 01:00",
+        ], [
+          'label' => 'Languages',
+          'body' => 'Estonian, French',
+        ],
+      ],
+    ];
+    $field_list_assert->assertPattern($field_list_expected_values, $practical_list_content->getOuterHtml());
+
+    // Assert Introduction field.
+    $node->set('oe_summary', 'Event introduction')->save();
+    $this->drupalGet($node->toUrl());
+    $page_header_expected_values['description'] = 'Event introduction';
+    $page_header_assert->assertPattern($page_header_expected_values, $page_header->getOuterHtml());
+
+    // Assert "Venue" field.
+    $venue_entity = $this->createVenueEntity('event_venue');
+    $node->set('oe_event_venue', [$venue_entity])->save();
+    $this->drupalGet($node->toUrl());
+
+    $field_list_expected_values = [
+      'items' => [
+        [
+          'label' => 'Where',
+          'body' => "event_venue\n  Address event_venue, 1001 Brussels, Belgium",
+        ], [
+          'label' => 'When',
+          'body' => "Wednesday 19 February 2020, 01:00\n to Friday 28 February 2020, 01:00",
+        ], [
+          'label' => 'Languages',
+          'body' => 'Estonian, French',
+        ],
+      ],
+    ];
+    if (version_compare(PHP_VERSION, '7.3') < 0) {
```

Comment:
As I understand that change based on that decision: https://github.com/openeuropa/oe_theme/pull/629#discussion_r531058337
Should we add here a comment for describing the problem and a link to a follow-up ticket for better tracking?

Reply:
Yes, based on the decision above and root of cause is explained here https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EWPP-437?focusedCommentId=4402494&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4402494

Reply:
Todos have been added. Also it is described in the https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-3356.